src/diff/index.js

```diff
@@ -227,19 +227,21 @@ export function diff(
  * @param {import('../internal').VNode} root
  */
 export function commitRoot(commitQueue, root) {
-	if (options._commit) options._commit(root, commitQueue);
-
-	commitQueue.some(c => {
-		try {
-			commitQueue = c._renderCallbacks;
-			c._renderCallbacks = [];
-			commitQueue.some(cb => {
-				cb.call(c);
-			});
-		} catch (e) {
-			options._catchError(e, c._vnode);
-		}
-	});
+	try {
+		if (options._commit) options._commit(root, commitQueue);
```

Comment:
does it make sense to skip all of them for one error? sorry not super familiar with internals right now

Reply:
It does make sense but we need the ref to our current vnode, will refactor to use one try block